__tests__/api.test.js

```diff
@@ -299,4 +319,42 @@ describe("GET /api/reviews where filter matches query", () => {
   });
 });
 
-describe("GET api/reviews where filter matches the last query", () => {});
+describe("GET api/reviews/:review_id/comments", () => {
+  it("should return 200: bad request when query name is incorrect", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(response => {
+        expect(response.body.comment).toEqual({
+          author: "bainesface",
+          body: "I loved this game too!",
+          comment_id: 1,
+          created_at: "2017-11-22T12:43:33.389Z",
+          review_id: 2,
+          votes: 16,
+        });
+      });
```

Comment:
I'm a bit confused. is it 200 or a bad request? from your test it looks like we are expecting to get one comment back